circuit/types/field/src/square_root.rs

```diff
@@ -63,6 +63,65 @@ impl<E: Environment> Field<E> {
     }
 }
 
+impl<E: Environment> Field<E> {
+    /// Returns both square roots of `self` and a `Boolean` flag, which is set iff `self` is not a square.
+    ///
+    /// If `self` is a non-zero square,
+    ///  - the first field result is the positive root (i.e. closer to 0)
+    ///  - the second field result is the negative root (i.e. closer to the prime)
+    ///  - the flag is 0
```

Comment:
```suggestion
    ///  - the first field result is the witnessed and checked square root
    ///  - the second field result is the negation of the first result
    ///  - the flag is 0.
    /// Since the standard console square_root function returns the positive square root
    /// (i.e. closer to 0), normally the first result would be less than the second result.
    /// However, that is not checked here and it would be a mistake to rely on the ordering.
```